lib/dialects/abstract/query-generator.js

```diff
@@ -177,7 +177,7 @@ class QueryGenerator {
         }
       }
     }
-
+    
```

Comment:
I've seen some of these changes here that add empty whitespace. Not sure why it is allowed by the linter, but could you maybe remove these?

Reply:
Done